indico/modules/events/templates/display/_manage_button.html

```diff
@@ -0,0 +1,130 @@
+{% from 'events/notes/_note.html' import render_toggle_button %}
+
+{% macro _render_note_editor(item) -%}
+    <li>
+        <a href="" class="notes-editor"
```

Comment:
Not so easy. Our dear legacy CSS won't properly style links without `href` attribute. And I don't want to risk styling old anchors (even though it seems unlikely something will break).